coalib/coala_main.py

```diff
@@ -100,34 +100,35 @@ def run_coala(console_printer=None,
     """
     all_actions_possible = provide_all_actions()
     apply_single = None
+
     if getattr(args, 'single_action', None) is not None:
         while True:
             for i, action in enumerate(all_actions_possible, 1):
                 console_printer.print(format_lines('{}'.format(
-                    action), symbol='['))
+                    action), symbol='[', line_nr=' ' + str(i) + '. '))
```

Comment:
this doesn't seem right. the closing square bracket will never be added this way

Reply:
The line_nr was for printing the option number on the screen.

Reply:
>Also seems strange, since you are doing changes to coala_main.py, which means coala should not be working with any tag, but it is.

also I think the `action` handles that perfectly fine.

Reply:
ok that's correct

Reply:
https://gitter.im/coala/coala?at=5a8ed24ba2194eb80d9b7aa6 the output with `-A` tag seems to be showing 1 as Do Nothing, can you decrement the indices by 1. Also a * is printed before a 0 and you have added just a space before str(i) so make sure its well indented 👍

Reply:
Yeah! I'll keep that in mind.

Reply:
@ishanSrt  Is the * necessary to be added to 0 for Do (N)othing. What if we open a new issue for it? :smile: